RecoMuon/MuonIdentification/plugins/MuonIdProducer.cc

```diff
@@ -750,6 +779,16 @@ bool MuonIdProducer::isGoodRPCMuon( const reco::Muon& muon )
   return ( muon.numberOfMatchedRPCLayers( reco::Muon::RPCHitAndTrackArbitration ) > minNumberOfMatches_ );
 }
 
+bool MuonIdProducer::isGoodME0Muon( const reco::Muon& muon )
+{
+  if(muon.track()->pt() < minPt_ || muon.track()->p() < minP_) return false;
+  //These "ExtraSoftMuons" may not be relevant, since there's no eta coverage below eta 1.5 anyway - the eta requirement should always fail
+   if ( addExtraSoftMuons_ && 
+	muon.pt()<5 && fabs(muon.eta())<1.5 && 
+	muon.numberOfMatchedME0Layers( reco::Muon::ME0HitAndTrackArbitration ) > 0 ) return true;
+   return ( muon.numberOfMatchedME0Layers( reco::Muon::ME0HitAndTrackArbitration ) >= minNumberOfMatches_ );
```

Comment:
this selection is not practical considering that ME0 is actually just one station